oauth-client-util/src/main/java/org/radarcns/oauth/OAuth2Client.java

```diff
@@ -89,66 +86,44 @@ public OAuth2AccessTokenDetails getAccessToken() throws TokenException {
         return currentToken;
     }
 
-    public OkHttpClient getHttpClient() {
+    public synchronized OkHttpClient getHttpClient() {
```

Comment:
if `httpClient` is an instance variable, removing synchronised here would be fine.